system-addon/content-src/components/Sections/_Sections.scss

```diff
@@ -0,0 +1,76 @@
+.sections-list {
+
+  .section-top-bar {
+    position: relative;
+
+    .section-title {
+      float: left;
+    }
+
+    .section-info-option {
+      float: right;
+    }
+
+    .info-option-icon {
+      background-image: url('assets/glyph-info-option-12.svg');
+      background-size: 12px 12px;
+      background-repeat: no-repeat;
+      background-position: center;
+      height: 16px;
+      width: 16px;
+      display: inline-block;
+    }
+
+    .section-info-option div {
+      visibility: hidden;
+      opacity: 0;
+      transition: visibility 0.5s, opacity 0.5s ease-in-out;
+    }
+
+    .section-info-option:hover div {
+      visibility: visible;
+      opacity: 1;
+    }
+
+    .info-option {
+      position: absolute;
+      background: $white;
+      border: solid 1px $faintest-black;
+      border-radius: $border-radius;
+      font-size: 13px;
+      color: $info-option-text-color;
+      line-height: 120%;
+      width: 320px;
+      right: 0;
+      top: 34px;
+      margin-top: -4px;
+      margin-right: -4px;
+      padding: 24px;
+      user-select: none;
+      -moz-user-select: none;
```

Comment:
It was in the test pilot version, but I've checked and I don't think Firefox supports it unprefixed